src/SignalR/clients/java/signalr/messagepack/signalr.client.java.messagepack.javaproj

```diff
@@ -21,37 +19,14 @@
     <JavaBuildFiles Include="@(Jars);@(PomFile)"/>
   </ItemGroup>
 
-  <Target Name="Restore" />
-
-  <PropertyGroup>
-    <PackDependsOn Condition=" '$(NoBuild)' != 'true' ">
-      $(PackDependsOn);
-      Build
-    </PackDependsOn>
-  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="../core/signalr.client.java.core.javaproj" />
+  </ItemGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.targets))\Directory.Build.targets" />
```

Comment:
Same nit as in src/SignalR/clients/java/signalr/core/signalr.client.java.core.javaproj